tools/tfm/bin_utils/imgtool_lib/keys.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2017 Linaro Limited
+# Copyright (c) 2017-2018, Arm Limited.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Cryptographic key management for imgtool.
+"""
+
+from __future__ import print_function
+from Crypto.Hash import SHA256
+from Crypto.PublicKey import RSA
+from Crypto.Signature import PKCS1_v1_5, PKCS1_PSS
```

Comment:
These functions are coming from the [pycrypto](https://pypi.org/project/pycrypto/) package, correct?

If so, pycrypto is not currently a [dependency](https://github.com/ARMmbed/mbed-os/blob/master/requirements.txt). Could you please either:
- Add it to the requirements.txt file and "pin" the correct major version Ex. `pycrypto>=2.6.1,<3.0` (or whatever is the correct lower bound of the version)
- OR use a module that already comes with Python, if available.

Reply:
From `pycryptodome`, I will add it to `requirements.txt`, I don't really know what should be the version range.
Also, all the files under `tools/tfm/bin_utils` were copied as-is from the TF-M repository, do you think they should be added to the importing process in #9653?

Reply:
If you expect them to be updated with new releases from the TF-M repository, then yeah probably can't hurt.

Reply:
@bridadan Actually [pycryptodome](https://pypi.org/project/pycryptodome/) should be used as there is an [issue ](https://github.com/dlitz/pycrypto/issues/238) on Windows with python 3.6 and pycrypto, I will add pycryptodome to `requirements.txt`.